### PR TITLE
Deploy using serverless and AWS Lambda.

### PR DESCRIPTION
**Pull request by chasdevs**
Deploy to AWS Lambda using the serverless package.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,7 @@ npm-debug.log*
 # Local Configs
 config/**/local*
 service_account_key.json
+
+# Serverless
```
Comment (Zugwalt) [inline .gitignore]: 🔥

File: bin/Handler.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { ScheduledEvent, Context, Handler } from 'aws-lambda';
+import logger from "../src/app/logger/logger";
+import MeetingTrafficLights from "../src/app/trafficLights/MeetingTrafficLights";
+import LifxBulbNetwork from "../src/app/bulbNetworks/lifx/LifxBulbNetwork";
+import CalendarService from "../src/app/calendar/CalendarService";
+import container from "../src/app/inversify.config";
```
Comment (Zugwalt) [inline bin/Handler.ts]: 👍

File: bin/Handler.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    const lights = new MeetingTrafficLights();
+
+    const lifxNetwork = container.get(LifxBulbNetwork)
+    lights.addNetwork(lifxNetwork)
+
+    const calendarService = container.get(CalendarService)
+    lights.setCalendar(calendarService)
+
+    return await lights.syncBulbs()
```
Comment (Zugwalt) [inline bin/Handler.ts]: Nice. Clean.

File: config/default.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ roomBulbLabelOverrides: {
 }
 
 timing:
-  meetingEndIntervalMinutes: 3
+  meetingEndIntervalMinutes: 2
```
Comment (Zugwalt) [inline config/default.yml]: wrapping up within 2 minutes of red. bold.

File: serverless.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+service:
+  name: meeting-room-traffic-lights
+
+provider:
+  name: aws
+  runtime: nodejs10.x
```
Comment (Zugwalt) [inline serverless.yml]: 🔥

File: serverless.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+functions:
+  syncBulbs:
+    handler: bin/Handler.handle
+    name: meeting-room-traffic-lights
+    description: Sets light bulb colors for meeting rooms.
+    events:
+      - schedule:
+          rate: rate(1 minute)
```
Comment (Zugwalt) [inline serverless.yml]: :+1:

File: src/app/trafficLights/MeetingTrafficLights.ts [hunk 7]
```diff
@@ -15,8 +15,8 @@ export default class MeetingTrafficLights {
     private _roomBulbMap: Map<Room, Bulb> = new Map<Room, Bulb>();
 
     //Timing
-    private _meetingEndIntervalMinutes: number = 5;
-    private _meetingWarningIntervalMinutes: number = 5;
+    private _meetingEndIntervalMinutes: number = config.get('timing.meetingEndIntervalMinutes');
+    private _meetingWarningIntervalMinutes: number = config.get('timing.meetingWarningIntervalMinutes');
```
Comment (Zugwalt) [inline src/app/trafficLights/MeetingTrafficLights.ts]: Good catch!
(I actually thought we were doing this)

File: config/default.yml (hunk 4 above)
Comment (chasdevs) [reply config/default.yml]: haha that was a tweak just from watching meetings happen in the room. People rolled out and the next meeting was wondering why it was still red.

**Pull request merged**